src/test/java/jp/ikedam/jenkins/plugins/updatesitesmanager/ManagedUpdateSiteJenkinsTest.java

```diff
@@ -98,14 +105,17 @@ private File getResource(String name) throws URISyntaxException, FileNotFoundExc
         }
         return new File(url.toURI());
     }
-    
+
+    @Test
     public void testDoPostBack() throws URISyntaxException, IOException, GeneralSecurityException
     {
         // CrumbIssuer causes failures in POST request.
         // I don't know why CrumbIssuer is enabled in a test environment...
         CrumbIssuer crumb = Jenkins.getInstance().getCrumbIssuer();
-        try
-        {
+
+        // By default it uses server
+        GlobalConfiguration.all().get(DownloadSettings.class).setUseBrowser(true);

```

Comment:
Note:
Jenkins gets to download update-center.json directly since 1.557.
https://issues.jenkins-ci.org/browse/JENKINS-19081

This code is to have Jenkins behave as < 1.557.